### PR TITLE
Gives Synx simplemobs more emotes (and scugs can wawa)

### PR DESCRIPTION
**Pull request by ReoDaProtovali**
## About The Pull Request
Synx are good at mimicking so they get all human emotes
Also catslugs can now wawa. No idea how we added a wawa emote without giving it to them!
## Changelog
:cl:
add: Added more emotes to synx
add: Added wawa to the wawa-ers
/:cl:

### CODE REVIEW

**Comment by ReoDaProtovali** [comment]
oh yeah it also implements synx voresprites

File: modular_chomp/code/modules/mob/living/simple_animal/aliens/synx.dm [hunk 1]
```diff
@@ -141,11 +146,14 @@
 	max_n2 = 0 //Maybe add a max
 	// TODO: Set a max temperature of about 20-30 above room temperatures. Synx don't like the heat.
 
+/mob/living/simple_mob/animal/synx/get_available_emotes()
+	. = ..()
+	. | _human_default_emotes //Synx are great at mimicking
```
Comment (Raeschen) [inline modular_chomp/code/modules/mob/living/simple_animal/aliens/synx.dm:151]: should be |= ?

File: modular_chomp/code/modules/mob/living/simple_mob/catslug.dm [hunk 2]
```diff
@@ -0,0 +1,4 @@
+//Scug go wawa
+/mob/living/simple_mob/animal/synx/get_available_emotes()
+	. = ..()
+	. | /decl/emote/audible/wawa
```
Comment (Raeschen) [inline modular_chomp/code/modules/mob/living/simple_mob/catslug.dm:4]: should be |= as well

**Comment by ReoDaProtovali** [comment]
There ya goes

**Pull request merged**